src/commands.ts

```diff
@@ -597,12 +617,6 @@ export class Commands {
 			}
 			folderPath = agent.expanded_directory;
```

Comment:
This would override any folder passed via query param, so I think this will have to be:

```suggestion
      if (!folderPath) {
        folderPath = agent.expanded_directory;
      }
```